### ISSUE TITLE
Portable Edition doesn't use user.config in AppData/Roaming

### ISSUE THREAD

**Issue by AlacrityF42** [mRemoteNG/mRemoteNG#414]
The Portable edition of mRemoteNG 1.75 RC1 saves user.config to both AppData\Local and AppData\Roaming. When you launch the software it doesn't look for AppData\Roaming, so it thinks you've never run the software before. This does not happen in version 1.72.

|Operating system | Windows 7 x64 |
|mRemoteNG version| 1.75 RC1  |

**Comment by sparerd** [comment]
Thanks for the report @AlacrityF42 , we will look into this.

**Comment by digitalexpl0it** [comment]
I am seeing the same issue, glade its not just me. I use roaming profiles and settings reset after each log off. Would be great if this could be fixed soon. Product has been a great tool and I appreciate the efforts on it.

**Comment by kmscode** [comment]
Comment from #488 
>mRemoteNG is installed on multi user environement (Terminal Server with Citrix Xenapp 6.5 in this case).
>User profile is configured to roam. In this configuration, appdata/local content will be deleted at logoff.
>Opening mRemoteNG, user.config stored (manually) in appdata/roamin will load correctly and all config are OK.
>If you edit configuration using Tools/Options a new user.config is created in appdata/local (using running configuration parameters) and not in appdata/roaming.
>Existing user.config stored originally in appdata/roamin will not be updated.
>At Operating System user logoff the file is deleted because all appdata/local and subfolders are discarded in TS environement with roaming profiles.

**Comment by kmscode** [comment]
Tentatively scheduled for 1.76

**Comment by crfiemme** [comment]
#488 refers to installed application not portable version ;-)
Thanks for linking #488 with this thread and sorry for duplicating.
Mirco

**Comment by kmscode** [comment]
@crfiemme 

While I haven't fully found the issue yet, most of these types of problems have code like this:
```
#if !PORTABLE
        public static readonly string SettingsPath = Do the non-portable thing;
#else
        public static readonly string SettingsPath = Do the portable thing;
#endif
```

So the fix will likely be "the same" - or at least right next to each other. And best to handle this with the one bug to ensure that they are both fixed at the same time.

**Comment by kmscode** [comment]
Reviewing this further, this might ultimately be resolved with #487

**Comment by sparerd** [comment]
@digitalexpl0it In #704 the portable version was changed to save program settings in the same directory as mremoteng.exe. Would you consider that a valid fix for the issues you are seeing? Or would you like us to keep this open to add the option to save settings to AppData\Roaming?

**Comment by AlacrityF42** [comment]
That works. Thanks!

-- Chuck

**Comment by sparerd** [comment]
Great! This will be available in v1.76
